CapabilityStatements/CapabilityStatement-mp-MedicationData.SendReceive.xml

```diff
@@ -17,7 +17,7 @@
       <use value="work"/>
     </telecom>
   </contact>
-  <description value="This CapabilityStatement describes the minimal requirements for a client/server to fulfill the 'Send/Receive medication data' transaction within Medication Process 9 (MP9) version 2.0.0. See [Landingspagina Medicatieproces](https://informatiestandaarden.nictiz.nl/wiki/Landingspagina_Medicatieproces) for additional information."/>
+  <description value="This CapabilityStatement describes the minimal requirements for a client/server to fulfill the 'Send/Receive medication data' transaction within Medication Process 9 (MP9) version 3.0.0-beta.2. See [Landingspagina Medicatieproces](https://informatiestandaarden.nictiz.nl/wiki/Landingspagina_Medicatieproces) for additional information."/>
```

Comment:
Is het handig de release zo specifiek te benoemen in de tekst? Want als beta.3 komt is er waarschijnlijk geen wijziging nodig aan het capability statement, het gaat dan verwarrend worden voor implementers als hier beta.2 staat. Versionering doen we toch via packaging, dan beter hier geen versies noemen (behalve misschien de major?)

Reply:
Je zou volgens mij op major- of minor-niveau moeten verwijzen, niet op de specifieke beta.

(En eigenlijk zou je naar het versienummer van de transactie moeten verwijzen, maar zo ver zijn we misschien nog niet).

Reply:
Voor nu verwezen naar major.minor-versie (dus 3.0.0). Mocht er op basis van de volgende sitdown een ander besluit worden genomen, dan kunnen we dit eventueel nog meenemen voor de beta.2-release, en anders doorvoeren bij de beta.3-release.